storybook.test.ts

```diff
@@ -1,4 +1,11 @@
 import initStoryshots from "@storybook/addon-storyshots";
 import "jest-styled-components";
 
-initStoryshots();
+initStoryshots({
+  asyncJest: true,
+  test: async ({ story, done }) => {
+    const jsx = await story.render();
+    expect(jsx).toMatchSnapshot();
+    done();
+  }
+});
```

Comment:
I think this could use a comment explaining where we drew inspiration for the code and why we're doing it this way.

Reply:
Good call! I sort of forgot myself since I made the commit. Have added a description in fb9af7236d2ac829377f8cb9cbe7d9a7ac34537e